Source/DafnyDriver/BoogieXmlConvertor.cs

```diff
@@ -24,33 +24,25 @@ namespace Microsoft.Dafny {
   /// </summary>
   public static class BoogieXmlConvertor {
 
-    public static void RaiseTestLoggerEvents(string fileName, string loggerConfig) {
-      string loggerName;
-      Dictionary<string, string> parameters;
-      int semiColonIndex = loggerConfig.IndexOf(";");
-      if (semiColonIndex >= 0) {
-        loggerName = loggerConfig[..semiColonIndex];
-        var parametersList = loggerConfig[(semiColonIndex + 1)..];
-        parameters = parametersList.Split(",").Select(s => {
-          var equalsIndex = s.IndexOf("=");
-          return (s[..equalsIndex], s[(equalsIndex + 1)..]);
-        }).ToDictionary(p => p.Item1, p => p.Item2);
-      } else {
-        loggerName = loggerConfig;
-        parameters = new();
-      }
-
-      // The only supported value for now
-      if (loggerName != "trx") {
-        throw new ArgumentException($"Unsupported verification logger name: {loggerName}");
-      }
-
-      // Provide just enough configuration for the TRX logger to work
-      parameters["TestRunDirectory"] = Constants.DefaultResultsDirectory;
+    public static TestProperty ResourceCountProperty = TestProperty.Register("TestResult.ResourceCount", "TestResult.ResourceCount", typeof(int), typeof(TestResult));
```

Comment:
We can just remove this and the references for now. When we do put it back, I'd prefer it wasn't inside the `BoogieXMLConvertor` namespace since we want to change the implementation of how the test events are raised later.

Reply:
Oh, right. I started to remove references to resource counts but didn't remove all of them. Thanks!